app/_hub/kong-inc/ai-proxy-advanced/how-to/machine-learning-platform-integrations/bedrock.md

```diff
@@ -0,0 +1,62 @@
+---
+nav_title: Bedrock
+title: Set up AI Proxy Advanced with Bedrock
+---
+
+This guide walks you through setting up the AI Proxy Advanced plugin with [Amazon Bedrock](https://aws.amazon.com/bedrock/).
+
+For all providers, the Kong AI Proxy plugin attaches to route entities.
+
+## Prerequisites
+* [{{site.base_gateway}} is installed and running](/gateway/latest/get-started/)
+* An AWS account
+* Any keys?
+
+## Configure the AI Proxy plugin
+
+1. Create a service in {{site.base_gateway}} that will represent the Bedrock API:
+```sh
+curl -X POST http://localhost:8001/services \
+ --data "name=bedrock-service" \
+ --data "url=https://bedrock-runtime.us-east-1.amazonaws.com"
+```
+
+1. Create a route that maps to the service you defined:
+```sh
+curl -i -X POST http://localhost:8001/routes \
+    --data "paths[]=/bedrock" \
+    --data "service.id=$(curl -s http://localhost:8001/services/bedrock-service | jq -r '.id')"
+```
+
+1. Use the Kong Admin API to configure the AI Proxy Advanced Plugin to route requests to Amazon Bedrock:
```

Comment:
For full disclosure, I used AI to help me fix my formatting for `targets` (tried to do it by hand, got some errors with {} etc.). But I also visually checked the output to make sure all the fields were there and then ran this example and the plugin successfully installed.